pipeline/src/pipeline.rs

```diff
@@ -51,8 +51,6 @@ pub struct ProofResult<T: FieldElement> {
     pub witness: Option<Vec<(String, Vec<T>)>>,
     /// Proof, potentially None (if success is false)
     pub proof: Option<Proof>,
-    /// Serialized low level constraints, potentially None (if success is false)
-    pub constraints_serialization: Option<String>,
```

Comment:
why?

Reply:
don't we need it for the tests?

Reply:
I hope not. I remember running all the tests last friday. This was just so we export the json to the prover-cpp.

Reply:
Yea it's tricky. I agree Pipeline shouldn't really need to know about this, but at the same time I also don't think a backend should be writing specific things to the disk.